.yarn/sdks/typescript/lib/tsserverlibrary.js

```diff
@@ -159,9 +152,6 @@ const moduleWrapper = tsserver => {
         typeof parsedMessage.arguments.hostInfo === `string`
       ) {
         hostInfo = parsedMessage.arguments.hostInfo;
-        if (hostInfo === `vscode` && process.env.VSCODE_IPC_HOOK && process.env.VSCODE_IPC_HOOK.match(/Code\/1\.([1-5][0-9]|60)\./)) {
-          hostInfo += ` <1.61`;
-        }
```

Comment:
Why is it not reliable?

Reply:
When using the Remote - WSL extension it doesn't exist, on Windows it's set to `\\\\.\\pipe\\<uuid>-1.66.0-main-sock`, and on Linux it's `/run/user/1000/vscode-dc336b01-1.61.2-main.sock`.

Reply:
I'd prefer if we could still support 1.65 on at least OSX - some engineers will purposefully avoid the upgrade until the fix is released, it'd be best to avoid breaking/punishing them if the extension auto-updates once the fix lands.

Reply:
In that case they would just not update the SDK, this doesn't modify the extension.

Reply:
I added a version check which works for both Windows and Linux, can't do much about Remote - WSL though.